sqlglot/expressions.py

```diff
@@ -3859,6 +3859,18 @@ def __init__(self, **args):
         super().__init__(**args)
 
 
+# https://www.oracletutorial.com/oracle-basics/oracle-interval/
+# https://trino.io/docs/current/language/types.html#interval-year-to-month
+class IntervalYearToMonthSpan(Expression):
+    arg_types = {}
+
+
+# https://www.oracletutorial.com/oracle-basics/oracle-interval/
+# https://trino.io/docs/current/language/types.html#interval-day-to-second
+class IntervalDayToSecondSpan(Expression):
+    arg_types = {}
+
```

Comment:
FYI the approach of creating expressions here will be useful if / when we decide to support Oracle's full syntax of these interval types, since they can also be parameterized with additional values:

```
INTERVAL '120-3' YEAR(3) TO MONTH
INTERVAL '11 10:09:08.555' DAY TO SECOND(3)
```